content/100-getting-started/index.mdx

```diff
@@ -40,14 +41,27 @@ Explore our products that make it easy to build and scale data-driven applicatio
 
 [**<SignalStream />Prisma Pulse**](/pulse) allows you to react to database changes with type-safe model streams in real-time.
 
-</BorderBox>
+[**<PrismaPostgres />Prisma Postgres**](/orm/overview/databases/prisma-postgres) is a managed PostgreSQL service that gives you an _always-on_ database pay-as-you-go pricing. Prisma Postgres is currently in [Early Access](/platform/maturity-levels#early-access).
```

Comment:
```suggestion
[**<PrismaPostgres />Prisma Postgres**](/orm/overview/databases/prisma-postgres) is a managed PostgreSQL service that gives you an _always-on_ database with pay-as-you-go pricing. Prisma Postgres is currently in [Early Access](/platform/maturity-levels#early-access).
```

Reply:
@sneub might have an idea also on the 1 sentence "what is PPG" messaging. Maybe we need some pre-made options in the value messaging doc? E.g. in 5 words, in 1 sentence, in 3 sentences, in a paragraph. Then we can keep reusing that.